database/src/main/scala/cromwell/database/slick/BackendKVStoreSlickDatabase.scala

```diff
@@ -0,0 +1,24 @@
+package cromwell.database.slick
+
+import cromwell.database.sql.BackendKVStoreSqlDatabase
+import cromwell.database.sql.tables.BackendKVStore
+
+import scala.concurrent.{ExecutionContext, Future}
+
+trait BackendKVStoreSlickDatabase extends BackendKVStoreSqlDatabase {
+  this: SlickDatabase =>
+
+  import dataAccess.driver.api._
+
+
+  override def addBackendJobKeyValuePair(entries: Iterable[BackendKVStore])(implicit ec: ExecutionContext): Future[Unit] = {
+    val action = dataAccess.backendKVStoreAutoInc ++= entries
+    runTransaction(action) map { _ => () }
```

Comment:
I realise I'm probably just barking into the wind, but my vote is for the non-scalaz version